benchmark.py

```diff
@@ -154,29 +154,29 @@ def _build_bazel_binary(commit, repo, outroot, platform=None):
   return the path without re-building.
 
   Args:
-    commit: the Bazel commit SHA.
+    commit: the commit, branch or tag to build Bazel at.
```

Comment:
In that case, should this parameter be changed to `version_identifier` or something, instead of `commit`?